src/applications/vaos/actions/newAppointment.js

```diff
@@ -110,6 +117,25 @@ export function startRequestAppointmentFlow() {
   };
 }
 
+export function openTypeOfCarePage(page, uiSchema, schema) {
+  return (dispatch, getState) => {
+    const state = getState();
+    const email = selectVet360EmailAddress(state);
```

Comment:
All the info should be in the profile when this page is loaded, so grab it and send it along in the action.